src/Extension/CoreExtension.php

```diff
@@ -1693,8 +1693,12 @@ function twig_array_batch($items, $size, $fill = null, $preserveKeys = true)
     return $result;
 }
 
-function twig_array_filter($array, $arrow)
+function twig_array_filter(Environment $env, $array, $arrow)
 {
+    if ($env->getExtension('\Twig\Extension\SandboxExtension')->isSandboxed() && !$arrow instanceof Closure) {
```

Comment:
I would suggest swapping both operands.
Also, can't the extension be missing?

Reply:
hmm, indeed. We might not have the SandboxExtension registered.

Reply:
good catch! fixed